disco/disco.go

```diff
@@ -233,8 +232,12 @@ func (d *Disco) processZkEvent(e *zk.Event) error {
 		if err != nil {
 			d.stateLog.Log(logkey.ZkEvent, e, logkey.DiscoService, serviceName, log.Err, err, "Unable to find parent")
 		} else {
-			d.stateLog.Log(logkey.DiscoService, service, "refreshing")
-			return service.refresh(d.zkConn)
+			d.stateLog.Log(logkey.DiscoService, serviceName, fmt.Sprintf("disco refresh of %s service", serviceName))
+			d.stateLog.Log(logkey.DiscoService, serviceName, fmt.Sprintf("State before refresh: %s", service))
```

Comment:
is there a way to combine the two by having "disco refresh of %s service" as logkey.message and "state before refresh" as logkey.desc?

Reply:
We can. I thought it's more readable this way.
Is there a performance impact with 2 calls?

Reply:
easy to debug that's all